how-to/workspace-platform-starter/client/src/framework/platform/interopbroker.ts

```diff
@@ -571,6 +571,12 @@ export function interopOverride(
 								error
 							);
 						}
+						if (!isEmpty(title)) {
+							// ensure no element tags are provided in the title
+							// we don't know how this information will be used
+							// and title hasn't come from the app directory
+							title = title.replace(/<[^>]*>?/gm, "");
```

Comment:
Same comment as the other one, can we move this to utils as a sanitiseString method

Reply:
Will leave this one as is to reduce the amount of changes but v15 (that can have changes) has this extracted into a util.

Reply:
ok